babel.config.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = {
+  presets: ['@babel/react', '@babel/preset-flow'],
+  ignore: ['third_party'],
+  plugins: [
+    '@babel/proposal-class-properties',
+    'syntax-trailing-function-commas',
```

Comment:
From @loganfsmyth:

>[This can be removed, since it has landed in the language official and is thus always enabled.](https://github.com/airbnb/babel-preset-airbnb/pull/37#discussion_r216164876)